src/shared/components/plots/MultipleCategoryBarPlotUtils.ts

```diff
@@ -98,6 +98,71 @@ export function sortDataByCategory<D>(
         }
     });
 }
+export function getSortedMajorCategories(
```

Comment:
can we have a comment explaining what is meant by "Major Category" ?

Reply:
![image](https://github.com/user-attachments/assets/8928818d-ccfd-4ef3-b8b9-c2ad71563444)


The `data `is defined in a way that it contains x-axis labels referred to as major categories. This functionality is used twice: once during the sorting of data and again when the getLabels function in MultipleCategoryBarPlot.tsx is called. Therefore, I created a separate function for that to enhance reusability.